notification/src/handler/notification/notification.handler.js

```diff
@@ -83,23 +94,32 @@ async function updatePaymentWithRepeater(
       )
       break
     } catch (err) {
-      if (err.body.statusCode !== 409)
-        throw new Error(
-          `Unexpected error during updating a payment with ID: ${currentPayment.id}. Exiting. ` +
-            `Error: ${JSON.stringify(serializeError(err))}`
-        )
+      if (err.statusCode !== 409)
+        throw new CommercetoolsError({
+          stack: err.stack,
+          message:
+            `Unexpected error during updating a payment with ID: ${currentPayment.id}. Exiting. ` +
```

Comment:
```suggestion
            `Unexpected error on payment update with ID: ${currentPayment.id}. ` +
```

Reply:
https://github.com/commercetools/commercetools-adyen-integration/pull/629/commits/6a553e7af08c1e8c86aaee23ac64ced8484ad171

Reply:
Thanks. There is still a confusing/scary part saying "Exiting" :) Do we need it or what was the intention?

Reply:
https://github.com/commercetools/commercetools-adyen-integration/pull/629/commits/808a041ded5fabf89b52a706298200067d7773fc